src/components/clusters/ClustersListToolbar.tsx

```diff
@@ -12,29 +14,36 @@ import {
   ToolbarFilterProps,
   SelectProps,
   TextInputProps,
+  ButtonVariant,
+  Spinner,
 } from '@patternfly/react-core';
 import { FilterIcon } from '@patternfly/react-icons';
 import { Cluster } from '../../api/types';
-import { CLUSTER_STATUS_LABELS } from '../../config';
+import { CLUSTER_STATUS_LABELS, routeBasePath } from '../../config';
+import ToolbarButton from '../ui/Toolbar/ToolbarButton';
+import { ResourceUIState } from '../../types';
+import { selectClustersUIState } from '../../selectors/clusters';
 
 export type ClusterFiltersType = {
   [key: string]: string[]; // value from CLUSTER_STATUS_LABELS
 };
 
-type ClustersFilterToolbarProps = {
+type ClustersListToolbarProps = {
   searchString: string;
   setSearchString: (value: string) => void;
   filters: ClusterFiltersType;
   setFilters: (filters: ClusterFiltersType) => void;
 };
 
-const ClustersFilterToolbar: React.FC<ClustersFilterToolbarProps> = ({
+const ClustersListToolbar: React.FC<ClustersListToolbarProps> = ({
   searchString,
   setSearchString,
   filters,
   setFilters,
 }) => {
   const [isStatusExpanded, setStatusExpanded] = React.useState(false);
+  const history = useHistory();
+  const clustersUIState = useSelector(selectClustersUIState);
```

Comment:
OCM uses skeleton when waiting for inital load of the cluster list. Do we want to be aligned with it too? @andybraren @nirfarkas

Reply:
@mareklibra I'm not sure what you mean by "skeleton", can you please elaborate?
@jtomasek I think that the search field should be wider to show the full text of what you can search for (name, ID, base domain). Currently, it ends with an ellipsis which is not so informative.

Reply:
@mareklibra If we decide to go for a skeleton for loading, I'd prefer to do it in a subsequent change
@nirfarkas skeleton is a form of displaying loading process https://www.patternfly.org/v4/components/skeleton
I'd prefer sticking to using the styling which PatternFly provides to avoid having to handle responsivenes etc. The content of the search box is a placeholder. I don't think it necessarily needs to always contain the full text. As part of this patch I've added a title attribute which displays the full search bar description on hover which I think is sufficient in informing the user how the search input can be used.

Reply:
@jtomasek ThanksX2
+1 for tooltip (:

Reply:
Sounds like [not a great one](https://patternfly.slack.com/archives/C2V002SMS/p1602693144007000?thread_ts=1602692094.005800&cid=C2V002SMS) 😞 - we can merge and adjust later.